docs/pages/reference/backends.mdx

```diff
@@ -798,6 +798,295 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use an
+[Athena](https://aws.amazon.com/athena/)-based audit log system that manages
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve high
+availability. The Athena back-end supports only one type of Teleport data, audit log
+events.
+
+The Athena audit logs back-end provides better scalability and searchability in
```

Comment:
Style nit, to simplify:

The Athena audit back-end is better at scale and search than DynamoDB.